src/BuildAnalyzer/BuildAnalyzer.h

```diff
@@ -17,6 +17,13 @@ namespace Microsoft { namespace Cpp { namespace BuildInsights {
 class BuildAnalyzer
 {
 public:
+    enum class TimeDisplayEnum
+    {
+        Nano,
+        Mili,
+        Sec,
+    };
```

Comment:
Do you mind renaming these enum values to `Nanoseconds`, `Milliseconds` (mind the double L) and `Seconds`?

Although they are part of an enum whose name includes `Time` and that could be enough to fill the rest of the name mentally, I'd prefer full names for readability.
Also, `<chrono>` defines `std::nano` and `std::milli` as `std::ratio` specializations while `std::chrono::nanosecond` or `std::chrono::millisecond` are `std::chrono::duration` specializations. Naming the enum with full names looks like a better analogy.

Reply:
Done.